lib/jsdom/browser/parser/html.js

```diff
@@ -11,15 +13,19 @@ const attributes = require("../../living/attributes");
 const nodeTypes = require("../../living/node-type");
 
 const serializationAdapter = require("../../living/domparsing/parse5-adapter-serialization");
+const {
+  CUSTOM_ELEMENT_REACTIONS_STACK, invokeCEReactions, lookupCEDefinition
```

Comment:
I don't think CUSTOM_ELEMENTS_REACTIONS_STACK should be capitalized in this way; it's just an object.